Index.html

```diff
@@ -0,0 +1,153 @@
+          </div>
+          <div class="title">
+            <h4>Teachers</h4>
+          </div>
+          <div class="description">
+            <p>We have trained proffessionals who are mostly examiners.</p>
+          </div>
+        </div>
+          <!-- Card two -->
+          <div class="card">
+            <div class="icon">
+              <i class="fa-solid fa-people-group"></i>
+            </div>
+          
+          <div class="title">
+            <h4>Campers</h4>
+          </div>
+          <div class="description">
+            <p>We have trained proffessionals who are mostly examiners.</p>
+          </div>
+        </div>
+          <!-- Card three -->
+          <div class="card">
+            <div class="icon">
+              <i class="fa-solid fa-book-open"></i>
+              </div>
+              <div class="title">
+                <h4>Courses</h4>
+              </div>
+              <div class="description">
+                <p>We have trained proffessionals who are mostly examiners.</p>
+              </div>
+            </div>
+              <!-- Card four -->
+              <div class="card">
+                <div class="icon">
+                  <i class="fa-solid fa-person-hiking"></i>
+                </div>
+                <div class="title">
+                  <h4>Extras</h4>
+                </div>
+                <div class="description">
+                  <p>We have trained proffessionals who are mostly examiners.</p>
+                </div>
+              </div>
+                <!-- Card five -->
+                <div class="card">
+                  <div class="icon">
+                    <i class="fa-solid fa-globe"></i>
+                   </div>
+                  <div class="title">
+                    <h4>Countries</h4>
+                  </div>
+                  <div class="description">
+                    <p>We have trained proffessionals who are mostly examiners.</p>
+                  </div>
+                </div>
+      </div>
+                <button type="button" class="reg">Register for Free</button>
+    </section>
+    <!-- Teachers info section -->
+    <section class="teachers-section" id="teachers-section">
+      <h2 class="facilitators">Facilitators</h2>
+      <div class="facilitators-persons" id="facilitators-persons"> 
+      </div>
+    </section>
+    <!-- Sponsors section  -->
+    <section class="sponsor-sections" id="sponsor-sections">
+      <h3>Patners</h3>
+      <hr class="line">
+      <div class="sponsors-container">
+        <img src="./Assets/SLALogo.png" alt="SLA logo">
+        <img src="./Assets/linkedin.jpg" alt="linkedin logo">
+        <img src="./Assets/youtube.jpg" alt="youtube logo">
+        <img src="./Assets/facebook.jpg" alt="facebook logo">
+        <img src="./Assets/google.jpg" alt="google logo">
+        <p>Hope Alive Foundation</p>
+      </div>
+    </section>
+  </main>
+  <footer> <p>&copy; HackingHerCode</p></footer>
```

Comment:
- [ ] As pointed out by the previous reviewer, please make sure the footer mimics the original design. The site logo should be on the left and the write-up on the right.

![image](https://user-images.githubusercontent.com/59397159/176675458-520f4169-9f58-4600-a13b-a6fe2730497d.png)